src/lib/bootloader/grub2_widgets.rb

```diff
@@ -299,15 +299,17 @@ def label
     def help
       # TRANSLATORS: TrustedGRUB2 is a name, don't translate it
       res = _("<p><b>Trusted Boot</b> will install TrustedGRUB2\n" \
-          "instead of regular GRUB2.</p>\n" \
-          "<p>It means measuring the integrity of the boot process,\n" \
+          "instead of regular GRUB2." \
+          "It means measuring the integrity of the boot process,\n" \
           "with the help from the hardware (a TPM, Trusted Platform Module,\n" \
-          "chip).</p>\n")
+          "chip).\n")
       if grub2.name == "grub2"
-        res += _("<p>First you need to make sure Trusted Boot is enabled in the BIOS\n" \
-          "setup (the setting may be named Security Chip, for example).</p>\n")
+        res += _(" First you need to make sure Trusted Boot is enabled in the BIOS\n" \
+          "setup (the setting may be named Security Chip, for example).\n")
       end
 
+      res += "<p>"
```

Comment:
Is that correct? Should it rather be a closing tag?

```suggestion
      res += "</p>"
```

Reply:
correct. It is the last entry, so it is not visible when I am checking it.